### PR TITLE
AST-2158 - Fix global default icon visibility

### PR DESCRIPTION
**Pull request by neerajmasai**
### Description

         1. Icon change and css fixed to position correctly.
         2. Icon color should change to light when dark color is selected and vice versa.
         3. Globe Color should not break when opacity is lowered or increased.
         4. Covered some corner cases to avoid any js errors.
### Screenshots

https://d.pr/i/VqiSH8
### Types of changes

         1. Icon change and css fixed to position correctly.
         2. Icon color should change to light when dark color is selected and vice versa.
         3. Globe Color should not break when opacity is lowered or increased.
         4. Covered some corner cases to avoid any js errors.

### Checklist:
- [x] My code is tested
- [ ] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/customizer/extend-custom-controls/src/common/astra-color-picker-control.js [hunk 1]
```diff
@@ -3,6 +3,88 @@ import { __ } from '@wordpress/i18n';
 import { Component } from '@wordpress/element';
 import { Dashicon, Button, ColorIndicator, TabPanel, GradientPicker, __experimentalGradientPicker, ColorPicker, SelectControl, ColorPalette } from '@wordpress/components';
 import { MediaUpload } from '@wordpress/media-utils';
+import styled from 'styled-components';
```
Comment (nathanrodrigues2111) [inline inc/customizer/extend-custom-controls/src/common/astra-color-picker-control.js:6]: Found Some issues with  @neerajmasai 
- When i hover it does not seem to be aligned properly : https://d.pr/i/D5p0SV
- Cant click on hover and normal for links section : https://d.pr/i/syZeBg
- Console errors : https://d.pr/i/3dFLSN

**Comment by neerajmasai** [comment]
there is a newer PR fixing this issue.
